resources/dicts/events/death/general.json

```diff
@@ -1849,7 +1849,7 @@
         ],
         "tags": ["all_lives"],
         "weight": 20,
-        "event_text": "m_c and r_c talked each other into walking onto the ice of the frozen pond that lay not far from camp. It was a dare at first, but when m_c didn't immediately fall in, r_c joined {PRONOUN/m_c/object}. The two cats playfully slipped around for a bit before the faint cracks reached their ears. By then, it was too late to move.",
+        "event_text": "m_c and r_c went to play together on the frozen pond. They never returned.",
```

Comment:
Last c_n heard, m_c and r_c were going out to play. They don't return, and the ice on a nearby pond is cracked...